pallets/msa/src/lib.rs

```diff
@@ -140,6 +140,11 @@ pub mod pallet {
 		/// The origin that is allowed to create providers via governance
 		type CreateProviderViaGovernanceOrigin: EnsureOrigin<Self::RuntimeOrigin>;
 
+		/// The origin that is allowed to approve recovery providers
+		type RecoveryProviderApprovalOrigin: EnsureOrigin<
+			<Self as frame_system::Config>::RuntimeOrigin,
+		>;
```

Comment:
nit: match syntax for `CreateProviderApprovalOrigin`
```suggestion
		type RecoveryProviderApprovalOrigin: EnsureOrigin<Self::RuntimeOrigin>;
```